eclair-core/src/test/scala/fr/acinq/eclair/router/RoutingSyncSpec.scala

```diff
@@ -302,12 +304,9 @@ object RoutingSyncSpec {
     outputIndex <- 0 to 1
   } yield ShortChannelId(block, txindex, outputIndex)).foldLeft(SortedSet.empty[ShortChannelId])(_ + _)
 
-  // this map will store private keys so that we can sign new announcements at will
-  val pub2priv: mutable.Map[PublicKey, PrivateKey] = mutable.HashMap.empty
```

Comment:
Why not use a `concurrent.Map`?

Reply:
I started with that, but I've done to much actor programming and can't get myself to share state anymore...